libraries/windows.rb

```diff
@@ -0,0 +1,29 @@
+require 'wmi-lite'
+
+module SCSI
+  module Windows
+    def self.scsi_devices
+      wmi = ::WmiLite::Wmi.new('root\CIMV2')
+      wmi.instances_of('Win32_DiskDrive').reduce(::Mash.new) do |result, disk|
+        host    = disk['scsibus']
+        channel = disk['scsiport']
+        target  = disk['scsitargetid']
+        lun     = disk['scsilogicalunit']
+
+        key = "#{host}:#{channel}:#{target}:#{lun}"
+        next result if host.nil?
+
+        result.merge!(key => ::Mash.new(
+          host:    disk['scsibus'],
+          channel: disk['scsiport'],
+          target:  disk['scsitargetid'],
+          lun:     disk['scsilogicalunit'],
```

Comment:
OPT: you could reuse your variables here

Reply:
Using disk['stuff'] makes all the assignments align nicely. Is your comment purely stylistic or something else ?